packages/compass-connections/src/components/connection-list/connection-menu.tsx

```diff
@@ -81,17 +79,19 @@ function ConnectionMenu({
       >
         <MenuItem
           data-testid="copy-connection-string"
+          glyph={<Icon glyph="Copy" />}
           onClick={() => {
             copyConnectionString(connectionString).finally(() => {
               setMenuIsOpen(false);
             });
           }}
         >
-          Copy Connection String
+          Copy connection string
```

Comment:
So it should always be lowercase to not wrap?

Reply:
It just happens to shave off a pixel or two which helps. But sentence-case is probably the way to go anyway. See the thread https://mongodb.slack.com/archives/G2L10JAV7/p1663158472156239